k8s/kube-base/cron-job-xdmod-openshift-prod.yaml

```diff
@@ -28,6 +25,11 @@ spec:
                   value: /data/xdmod_openshift_data/nerc_openshift_prod
                 - name: XDMOD_RESOURCE
                   value: nerc_openshift_prod
+                - name: OPENSHIFT_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      key: token
+                      name: xdmod-openshift-prod-secret
```

Comment:
Why the change from `envFrom`? When you create the secret, you could just create it with an `OPENSHIFT_TOKEN` key. Either solution is fine; I'm just curious what motivated this particular change.

Reply:
Oh uh it's because all the examples in the documentation did it this way :)